src/sentry/models/grouphashmetadata.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_model
+
+
+@region_silo_model
+class GroupHashMetadata(Model):
+    __relocation_scope__ = RelocationScope.Excluded
```

Comment:
Do we want this?

Reply:
Yes, I believe so. My understanding of the migration system is that while things like org members and projects and teams travel with you, your event/group/etc data does not. Also, the `GroupHash` table [is excluded](https://github.com/getsentry/sentry/blob/a0a0832ef9fab0a962a2276339ab09f346317a07/src/sentry/models/grouphash.py#L14-L15), so it stands to reason the associated metadata must be as well.